behat.yml.dist

```diff
@@ -7,6 +7,9 @@ default:
         - Drupal\DrupalExtension\Context\MinkContext
         - Drupal\DrupalExtension\Context\DrupalContext
         - Drupal\Tests\oe_content\Behat\FeatureContext
+        - OpenEuropa\Behat\TransformationContext:
+            pages:
+              page creation: 'node/add/oe_page'
```

Comment:
Let's call this the `Page creation` so that the Behat step can be `And I visit "the Page creation page"`